drivers/hmis/lib/form_data/assessments/dummy_intake_assessment.json

```diff
@@ -281,17 +282,20 @@
               "type": "CHOICE",
               "link_id": "4.02.2",
               "text": "Income from Any Source",
-              "pick_list_reference": "NoYesReasonsForMissingData"
+              "pick_list_reference": "NoYesReasonsForMissingData",
+              "field_name": "incomeFromAnySource"
             },
             {
               "type": "CURRENCY",
               "link_id": "4.02.A",
-              "text": "Earned Income (i.e., employment income)"
+              "text": "Earned Income (i.e., employment income)",
+              "field_name": "earnedAmount"
```

Comment:
@eanders @awisema 
I didn't finish filling in all the field names here, figured I should get feedback before doing that repetitive task.

The HUD spec is simple enough that this approach should work for populating any HUD field into a form.

The frontend needs to graphql field name (eg `earnedAmount` not `earned_amount` because it's filling it based on the field on the record.

On the frontend, this works pretty much exactly the same way as "Edit Project" and "Edit Client" screens (also "Advanced Search"). Each of those takes a form definition and sets the initial form state for any `field_name`s (fka `query_field`) specified in the form definition.

I think we could also use this for processing, but we don't need to.